draft-ietf-httpbis-digest-headers.md

```diff
@@ -253,9 +254,11 @@ STRUCTURED-FIELDS}}):
 Representation-Digest   = sf-dictionary
 ~~~
 
-`Representation-Digest` member-keys convey the hashing algorithm (see
-{{algorithms}}) used to compute the digest. Member-values are the output
-of the digest calculation. Member-values MUST be of type sf-binary.
+where member-keys convey the hashing algorithm (see
+{{algorithms}}) used to compute the digest
+and have no parameters.
+Member-values are the output of the digest calculation
+and their syntax is `sf-binary`.
```

Comment:
Separately I'm not a big fan of saying "the syntax is sf-xyz" because it puts a lot of attention on syntax, which doesn't matter, and it doesn't put a normative requirement on the type, which is very important. 

How about "Member-values are the output of the digest calculation, the type MUST be Byte Sequence."?

This comment applies to other instances too.

Reply:
I  thought that syntax and type were tighly coupled, but if it's not the case (eg. a type can have different syntaxes)
it's ok.

As a follow up to your email, I think that a formal way to express relations between items - that is defining a "schema" for structured fields - would be great, but I suspect that it's not trivial. Moreover this could have sense only if fields are serialized differently from rfc8941.